internal/service/alertconfiguration/resource_test.go

```diff
@@ -566,6 +570,41 @@ func TestAccConfigRSAlertConfiguration_withVictorOps(t *testing.T) {
 	})
 }
 
+func TestAccConfigRSAlertConfiguration_withSeverityOverride(t *testing.T) {
+	var (
+		projectID = acc.ProjectIDExecution(t)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithSeverityOverride(projectID, conversion.StringPtr("ERROR")),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "severity_override", "ERROR"),
+					// Data source checks
+					checkExists(dataSourceName),
+					resource.TestCheckResourceAttr(dataSourceName, "severity_override", "ERROR"),
+				),
+			},
+			// todo: should check for no attr once CLOUDP-353933 is fixed
```

Comment:
```suggestion
			// TODO: Should check for no attr once CLOUDP-353933 is fixed.
```

nit: TODO uppercase and capitalized sentences with ending period.